File: C/level1/3 - Calculs et découverte des variables/11 - Course avec les enfants.c
//coded by cybereagle2001
#include <stdio.h>
#include <robot.h>

int nb=0;
int main()
{

   for (int _loop=1 ,_max=10;_loop<= _max;_loop++)
   {
      nb = nb+1;
      for (int _loop1= 1,_max1=nb;_loop1<= _max1;_loop1++)
      {
         droite();
      }
      ramasser();
      for (int _loop1= 1,_max1=nb;_loop1<= _max1;_loop1++)
      {
         gauche();
      }
      deposer();
   }
   return 0;
}

File: java/level 1/3- Calculs et découverte des variables/11- Course avec les enfants.java
/** coded by cybereagle2001 */

import static algorea.Robot.*;

class Main{
   public static void main(String[] args){
      for (int loop=1; loop<=10; loop++){
         for (int i=0; i< loop; i++){
            droite();
         }
         ramasser();
         for (int i=0;i<loop;i++){
            gauche();
         }
         deposer();
      }
   }
}    
